src/xdocs/cmdline.xml.vm

```diff
@@ -96,26 +96,27 @@ java -D&lt;property&gt;=&lt;value&gt;  \
           <code>-s</code> option. Default value is 8.
         </li>
         <li>
-          <code>-t, --tree</code> - Prints Abstract Syntax Tree(AST) of the checked file. The option
-          cannot be used other options and requires exactly one file to run on to be specified.
+          <code>-t, --tree</code> - This option is used to display the Abstract Syntax Tree (AST)
+          of the specified file. It can only be used on a single file and cannot be combined
+          with other options.
         </li>
         <li>
-          <code>-T, --treeWithComments</code> - Prints Abstract Syntax Tree(AST) with comment nodes
-          of the checked file. The option cannot be used with other options and requires exactly
-          one file to run on to be specified.
+          <code>-T, --treeWithComments</code> - This option is used to display the Abstract Syntax
+          Tree (AST) <b>with comment nodes</b> of the specified file. It can only be used on a
```

Comment:
Aren't javadoc still comments? It may sound like there is a chance for confusion here.

Reply:
We used term ` with Javadoc nodes ` for javadoc and `with comment nodes` for non-javadoc.

Should we add the word "non-javadoc" here?

Reply:
Done